org.eclipse.sisu.plexus/build.target

```diff
@@ -1,138 +1,260 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<?pde version="3.7"?>
 <target name="org.eclipse.sisu.plexus" sequenceNumber="0">
```

Comment:
Eclipse 2022-03 Target Editor generates those nasty new lines and wrong indentations. Please let me know if you want me to manually fix it.

Reply:
Tracked now in https://github.com/eclipse-pde/eclipse.pde/issues/253.